### DISCUSSION TITLE
New TRV Thermostat Card swaps set and current temperatur

### DISCUSSION THREAD

**Discussion by ehtron** [home-assistant/frontend#19038] [Dashboard]
Checklist

    I have updated to the latest available Home Assistant version.
    I have cleared the cache of my browser.
    I have tried a different browser to see if it is related to my browser.

Hi :)
the new thermostat card shows the temperature at the wrong point . here Shelly TRV

the current temperature should be displayed large in the center and the target temperature smaller below it.
as it was in the previous version.

this makes it easier to recognize the current temperature.
old
![grafik](https://user-images.githubusercontent.com/37932495/289361667-f79b91c6-efca-454a-ad3e-78f2cd21cd6b.png)

new 
![grafik](https://user-images.githubusercontent.com/37932495/289361717-b8e782c0-f6d0-43f3-bf45-77aaab07063c.png)

btw.
it is not a good idea to change the dimensions of an object afterwards so that the design is torn apart.
with the new height, it no longer fits on my tablet :(

regads 
ehtron

**Comment by karwosts** [comment]
I believe this is a subjective design decision, and not a bug, and as such will move this to the discussions area.
